Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/ComponentConfig.purs

```diff
@@ -471,6 +472,20 @@ rentalRateCardConfig state =
   in rentalRateCardConfig'
 
 
+autoAnimConfig :: AnimConfig.AnimConfig
+autoAnimConfig =
+  let
+    config = AnimConfig.animConfig
+
+    autoAnimConfig' =
+      config
+        { duration = 400
+        , toScaleX = 1.0
+        , toScaleY = 1.0
+        }
+  in
+    autoAnimConfig'
+
```

Comment:
here let and in is not needed. we can directly override the values

autoAnimConfig :: AnimConfig.AnimConfig
autoAnimConfig = AnimConfig.animConfig
        { duration = 400
        , toScaleX = 1.0
        , toScaleY = 1.0
        }